app/src/main/java/org/literacyapp/chat/ChatActivity.java

```diff
@@ -63,6 +66,29 @@ protected void onStart() {
         arrayAdapter = new MessageListArrayAdapter(getApplicationContext(), textMessages);
         mListPreviousMessages.setAdapter(arrayAdapter);
 
+        mTextMessage.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+            }
+
+            @Override
+            public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+            }
+
+            @Override
+            public void afterTextChanged(Editable editable) {
+
+                if(editable.toString().trim().length() > 0){
+                    mButtonSend.setColorFilter(Color.GREEN);
+                } else {
+                    mButtonSend.setColorFilter(Color.BLACK);
```

Comment:
If we use `mButtonSend.setEnabled(false);` instead, would that give the button the same effect of a gray color using less code?